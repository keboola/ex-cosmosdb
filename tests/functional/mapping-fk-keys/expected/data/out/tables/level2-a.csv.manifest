{
    "columns": [
        "count",
        "itemId",
        "level1_pk"
    ],
    "primary_key": [],
    "incremental": false
}
