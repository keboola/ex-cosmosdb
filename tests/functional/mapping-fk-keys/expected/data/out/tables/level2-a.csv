"1","123","be5e25e72e6b2a5454dc99f9cc830c21"
"1","123","8548ce314658dfbc022ca94635cf1905"
"20","456","177bc26b5ae19099c845ed74d094eed6"
"20","456","cc281168d52b3c39ad594a56c94548af"
